applications/TheLostDiamond_BjornOmsland.md

```diff
@@ -0,0 +1,208 @@
+- **Registered Address:** Rosenholmveien 25, 1414 Trollåsen, Norway
+
+### Team Experience
+
+We are four creative outdoor enthusiasts from Norway with backgrounds in the travel industry, logistics, project management and programming.
+
+We have hosted several Lost Diamond events in Norway, utilizing the EOS-Mainnet, for more than 1,000 users to test and improve TheLostDiamond race.
+
+Our programmer (Bjørn Omsland) has twenty years experience as a full stack developer and four year experience as a smart contract developer on the EOS-blockchain. 
+
+
+### Team Code Repos
+
+- https://github.com/bjornomsland
+- https://github.com/bjornomsland/CaptainBlackBillSmartContract
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+N/A
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/bjornomsland/
+- https://www.linkedin.com/in/ann-magrit-monhof-55785610/
+- https://www.linkedin.com/in/jorn-balslev/
+- https://www.linkedin.com/in/sivbalslev/
+
+## Development Status
+
+We have a working MVP that has been used for testing the application in several single-event races in Norway. https://cptblackbill.com/
+
+The smartcontract is located on the EOS-Mainnet account cptblackbill. The code is available at https://github.com/bjornomsland/CaptainBlackBillSmartContract
+
+The frontend is developed in NodeJs and hosted on Google Clouds. 
+
+Different single-event races can have different unique domain names. E.g. our biggest single-event race is “Discover Oslo Adventure Race”. We will host our third Discover Oslo Adventure Race on June 11th. https://opplevoslo.app The feedback from our Discover Oslo participants has been exceptionally good. 98% say they will participate in the upcoming Discover Oslo race.
+
+Development, testing and feedback from all the single-event races is the foundation for our main goal - launching The Lost Diamond 24/7 Global Adventure Race. The Lost Diamond 24/7 Global Adventure Race is ready for Test Marketing, where the goal is to find out how many people will participate in a pure crypto-based adventure race and see if we can obtain a 22% monthly growth rate. 
+
+
+## Development Roadmap
+
+**Marketing Roadmap**
+
+The first milestone (2 months duration) is to get a baseline of 100 non-tech people paying a $7 dollar monthly subscription fee to participate in The Lost Diamond adventure race. 
+
+### Overview
+
+- **Total Estimated Duration:** 2 months (for the first milestone)
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 10,000 USD (for the first milestone. The whole project will take two years and the estimated total need for funding is $500,000 USD)
+
+### Milestone 1 — Prove that TheLostDiamond MVP can get a baseline of 100 participants and obtain a 22% monthly growth rate.
+
+- **Estimated duration:** 2 months
+- **FTE:**  1
+- **Costs:** 10,000 USD
```

Comment:
First milestone's budget is going towards solely "marketing" purpose.

If none of the funding is going towards development, how is the team currently being funded to develop this application?

Funds received by a grant should be allocated towards development purposes.

Reply:
We have developed this application with no funding since 2018. The smart contract is open source and available at https://github.com/bjornomsland/CaptainBlackBillSmartContract/blob/master/src/cptblackbill.cpp

Getting funding for EOS projects has not been easy over the past years. I think it's important to also allow funding to projects that has been developing on EOS for several years. Our MVP is ready and next step is to prove that we can get traction. That is why we only need funding for marketing purpose.